### ISSUE TITLE
v0.25.0rc.2 CORS issues

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/MeiliSearch#2040]
**Describe the bug**
CORS issue when making request from the browser: 

```
Access to fetch at 'http://127.0.0.1:7700/indexes/createdIndexTest' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**To Reproduce**
1. Run a meilisearch
2. Create an index.html
3. Request on `GET /indexes`

**Expected behavior**
No Cors issues

**MeiliSearch version:** v0.25.0rc2

**Additional context**
Browser: Chrome

**Comment by untitledpng** [comment]
Hi @bidoubiwa !
I'm no expert at MeiliSearch so take my answer with a little grain of salt ;). But every time I got the CORS error in my browser when performing a request to MeiliSearch, It was because I was not including the `X-Meili-API-Key` header with the public key. (Or I had a typo in the key)

My advice is to check it one more time just to be sure! :)

Happy holidays!

**Comment by curquiza** [comment]
Thanks @bidoubiwa, we will fix that for v0.25.0

**Comment by curquiza** [comment]
Fixed thanks to #2050
